package.json

```diff
@@ -37,25 +37,27 @@
   },
   "devDependencies": {
     "@axe-core/react": "^4.10.0",
+    "@eslint/eslintrc": "^3.1.0",
```

Comment:
I wonder if this `eslintrc` package is needed? Looking at the GitHub page, it provides some legacy configuration and is not under active development https://github.com/eslint/eslintrc

Reply:
True that `@eslint/eslintrc`can now be removed as the migration is done and the `FlatCompat` is now longer needed. Will remove that. Nice find.

Reply:
`eslint/js` is used in `eslint.config.js` to get the recommended tules for eslint.

It is still used even if we are using `typescript-eslint`.

At least that is my understanding from the documentation, where typescript stuff is in addition.
https://typescript-eslint.io/getting-started#step-2-configuration

For example the `no-console` warn comes from that `eslint/js` rule as there is different rules for `@typescript-eslint`